models/surgical_tool_segmentation/configs/train.json

```diff
@@ -0,0 +1,274 @@
+        "lr": 0.0001
+    },
+    "train": {
+        "deterministic_transforms": [
+            {
+                "_target_": "LoadImaged",
+                "keys": [
+                    "image",
+                    "label"
+                ]
+            },
+            {
+                "_target_": "ToTensord",
+                "keys": [
+                    "image",
+                    "label"
+                ]
+            },
+            {
+                "_target_": "AsChannelFirstd",
+                "keys": "image"
+            },
+            {
+                "_target_": "AddChanneld",
+                "keys": "label"
+            },
+            {
+                "_target_": "Resized",
+                "keys": [
+                    "image",
+                    "label"
+                ],
+                "spatial_size": [
+                    736,
+                    480
+                ],
+                "mode": [
+                    "bilinear",
+                    "nearest"
+                ]
+            },
+            {
+                "_target_": "ScaleIntensityd",
+                "keys": [
+                    "image",
+                    "label"
+                ]
+            }
+        ],
+        "random_transforms": [
+            {
+                "_target_": "RandRotated",
+                "keys": [
+                    "image",
+                    "label"
+                ],
+                "range_x": "$np.pi",
+                "prob": 0.8,
+                "mode": [
+                    "bilinear",
+                    "nearest"
+                ]
+            },
+            {
+                "_target_": "RandZoomd",
+                "keys": [
+                    "image",
+                    "label"
+                ],
+                "min_zoom": 0.8,
+                "max_zoom": 1.2,
+                "prob": 0.2,
+                "mode": [
+                    "bilinear",
+                    "nearest"
+                ]
+            }
+        ],
+        "preprocessing": {
+            "_target_": "Compose",
+            "transforms": "$@train#deterministic_transforms + @train#random_transforms"
+        },
+        "dataset": {
+            "_target_": "CacheDataset",
+            "data": "$[{'image': i, 'label': l} for i, l in zip(@images, @labels)]",
+            "transform": "@train#preprocessing",
+            "cache_rate": 0.1,
+            "num_workers": 4
+        },
+        "dataloader": {
+            "_target_": "DataLoader",
+            "dataset": "@train#dataset",
+            "batch_size": 8,
+            "shuffle": true,
+            "num_workers": 4
+        },
+        "inferer": {
+            "_target_": "SimpleInferer"
+        },
+        "handlers": [
+            {
+                "_target_": "ValidationHandler",
+                "validator": "@validate#evaluator",
+                "epoch_level": true,
+                "interval": 1
```

Comment:
Hi @binliunls , please define a variable called `val_interval`, see also: https://github.com/Project-MONAI/model-zoo/blob/1912aba5a6211c9b49510735180fbecf6489131c/CONTRIBUTING.md#preferred-files-and-keys
It is a requirement from monai FL side.

Reply:
Thanks @yiheng-wang-nv , I have added the val_interval parameter to the train.json